packages/playground/website/src/lib/site-storage.ts

```diff
@@ -22,7 +23,8 @@ const SITE_METADATA_FILENAME = 'playground-site-metadata.json';
  * NOTE: We are using different storage terms than our query API in order
  * to be more explicit about storage medium in the site metadata format.
  */
-export type SiteStorageType = 'temporary' | 'opfs' | 'local-fs';
+export const SiteStorageTypes = ['opfs', 'local-fs', 'none'] as const;
```

Comment:
I don't have strong feelings between using 'temporary' or 'none', but when I think about the possibility of persisting even temporary sites to OPFS as a safety measure against lost data (due to things like browser crash, power outage, or forced reboot), then it seems like it would be clearer for those short-term site configs to be tagged with `"storage": "temporary"` rather than `"storage": "none"`.

Reply:
How about `"storage": "in-memory"`? I'm happy with either.

Reply:
I like it! "in-memory" is precise.